internal/transform/ledger.go

```diff
@@ -55,6 +55,12 @@ func TransformLedger(inputLedger historyarchive.Ledger) (LedgerOutput, error) {
 
 	outputProtocolVersion := uint32(ledgerHeader.LedgerVersion)
 
+	var outputSorobanFeeWrite1Kb int64
+	lcmV1, ok := lcm.GetV1()
+	if ok {
+		outputSorobanFeeWrite1Kb = int64(lcmV1.Ext.V1.SorobanFeeWrite1Kb)
```

Comment:
New fee change

Reply:
may be worthwhile to use the safety getter method of `LedgerCloseMetaExt.GetV1()` rather than direct reference to nested `V1` struct, to avoid potential nil pointer,  i.e. check `ok` from `lcmv1.Ext.GetV1()`

Reply:
Makes sense. Done in  [d43e728](https://github.com/stellar/stellar-etl/pull/240/commits/d43e728fe285fd51bd2cd30fb18e5c6634bff6a6)